templates/telco/osm/onboarding-network-functions.html

```diff
@@ -0,0 +1,187 @@
+      <p>Using private and public cloud infrastructure at the same time allows you to optimise your CapEx and OpEx costs.</p>
+    </div>
+		<hr style="margin-bottom: 2rem;" />
+		<div class="col-12 u-hide--small u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/68971b5f-Onboarding+3.svg",
+        alt="",
+        width="1000",
+        height="500",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+			<h2>VNF onboarding in OSM Hackfests</h2>
+      <p>Canonical regularly participates in the events organized by the OSM community and ETSI’s Centre for Testing and Interoperability. For events like Hackfests and NFV Plugtests Canonical also offers a partner cloud where multiple VNF vendors can deploy and test the interoperability of their Telco network functions. It is also available if VNF vendors want to test their network functions individually before migrating to production environments.</p>
+    </div>
+    <div class="col-12">
+			Network Functions onboarded graph
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--suru-accent is-dark">
+  <div class="row">
+    
+    <div class="col-12">
+      <div class="p-heading-icon--small">
+        <h2>Tell us about your network function</h2>
+        <p>Canonical can help you automate your network function onboarding to Charmed OSM. Please choose the relevant options for your network functions to see how it can work for you.</p>
+        <p><a href="https://charmed-osm.com/#get-in-touch" class="p-button--positive">Start now</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+		<h2>Let us help you in onboarding</h2>
+    <div class="col-6 p-card">
+      <h4><a class="p-link--external" href="https://charmed-osm.com/">Charmed OSM deployment support</a></h4>
+			<hr />
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Supported with <a href="/ua">Ubuntu Advantage</a></li>
+        <li class="p-list__item is-ticked">Availability of managed and professional services</li>
+        <li class="p-list__item is-ticked">ESM - Security updates for Ubuntu LTS for additional 3-5 years</li>
+      </ul>
+    </div>
+    <div class="col-6 p-card">
+      <h4>Network function onboarding support </h4>
+			<hr />
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Support for automation of the network function</li>
+        <li class="p-list__item is-ticked">Onboarding it to Charmed OSM</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2>Learn onboarding from scratch</h2>
+    </div>
+    <div class="col-6 u-embedded-media u-hide--small">
```

Comment:
Why hide this on small screens?

Reply:
We seem to hide most images in small screens which I found it kind of odd. But yes, I guess this is a video shouldn't have this class